code/datums/actions/order_action.dm

```diff
@@ -119,6 +119,7 @@
 	var/skill_name = "leadership"
 	///What minimum level in that skill is needed to have that action
 	var/skill_min = SKILL_LEAD_EXPERT
+	keybind_signal = COMSIG_KB_RALLYORDER
```

Comment:
old vars need to be declared befor new one (in that cas you need to put that line 119)